### PR TITLE
[docs] mention a way to set the default toolchain for rust

### PR DESCRIPTION
**Pull request by JesterOrNot**
Mention the `rustup-toolchain` file :octocat:

### CODE REVIEW

File: src/docs/rust-in-gitpod.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ To fix this you'll need to wrap the `rustup` command in a login shell, like so:
 RUN bash -cl "rustup toolchain install nightly"
 ```
 
+You can override this default by commiting a `rustup-toolchain` file in the root of your repository. Sat for example you need the nightly compiler for March 4th 2020 you would add the `rustup-toolchain` file and add `nightly-2020-03-04` which will set the required Rust verion
```
Comment (nisarhassan12) [inline src/docs/rust-in-gitpod.md]: Thanks for taking care of this. Please add a . at the end of the paragraph.

Comment (JesterOrNot) [reply src/docs/rust-in-gitpod.md]: Done!

File: src/docs/rust-in-gitpod.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ To fix this you'll need to wrap the `rustup` command in a login shell, like so:
 RUN bash -cl "rustup toolchain install nightly"
 ```
 
+You can override this default by commiting a `rustup-toolchain` file in the root of your repository. Sat for example you need the nightly compiler for March 4th 2020 you would add the `rustup-toolchain` file and add `nightly-2020-03-04` which will set the required Rust verion.
```
Comment (nisarhassan12) [inline src/docs/rust-in-gitpod.md]: `Say` not 'Sat'

**Comment by JesterOrNot** [comment]
Thanks!

**Pull request merged**